Spreadsheet/style.css

```diff
@@ -41,20 +41,19 @@ header {
 .main-top {
     display: flex;
     align-items: center;
-    border: 1px solid #01A9B4;
+    border-color: #01A9B4;
+    border-style: solid;
     border-width: 1px 0;
```

Comment:
It's usually best to avoid overriding rules in the same selector. It technically works but makes reading the code a lot less straightforward. Best to separate it into individual rules rather than use the shorthand rule in this specific case.